docs/source/cpp/parquet.rst

```diff
@@ -571,6 +571,19 @@ More specifically, Parquet C++ supports:
 * EncryptionWithFooterKey and EncryptionWithColumnKey modes.
 * Encrypted Footer and Plaintext Footer modes.
 
+Field Id
+----------
+
+The parquet format supports an optional integer field_id which can be assigned
+to a field. This is used in the `iceberg specification <https://github.com/apache/iceberg/blob/main/format/spec.md#column-projection>` __
+
+On writer side, If ``PARQUET:field_id`` is present as a metadata key on a field, and the corresponding
+value is a nonnegative integer, then it will be used as the field_id in the parquet
```

Comment:
```suggestion
value is a nonnegative integer, then it will be used as the field id in the parquet
```

Reply:
Aha, the parquet thrift uses "field_id" here, any word is ok for me though